src/app/requirements.txt

```diff
@@ -1,3 +1,4 @@
+stomp
```

Comment:
I don't like the placement of this. Let's get a version locked in for this and move it down with the others.

Reply:
This doesn't work. You need to use `stomp.py` not `stomp`